sigtool/sigtool.c

```diff
@@ -326,6 +326,97 @@ static int hashsig(const struct optstruct *opts, unsigned int class, int type)
     return 0;
 }
 
+static int fuzzy_img_file(char *filename)
+{
+    int status = -1;
+
+    int target_fd                   = -1;
+    FFIError *fuzzy_hash_calc_error = NULL;
+    uint8_t *mem                    = NULL;
+
+    image_fuzzy_hash_t hash = {0};
+    STATBUF st;
+    ssize_t bytes_read;
+
+    if ((target_fd = open(filename, O_RDONLY)) == -1) {
+        mprintf(LOGG_ERROR, "%s: Can't open file\n", basename(filename));
```

Comment:
These messages seem to lack the error message (ala `strerror()`)-- is that handled elsewhere?

Reply:
Hmm yeah I should add the `strerror(errno)` output.